src/bundle/Controller/ContentController.php

```diff
@@ -331,4 +333,57 @@ public function previewAction(
             'versionNo' => $versionNo ?? $content->getVersionInfo()->versionNo,
         ]);
     }
+
+    /**
+     * @param Request $request
```

Comment:
Please use FQCN in doc blocks. Please fix other cases as well as I notice it in other files too.